test/doctests.jl

```diff
@@ -1,9 +1,11 @@
 using Test, Documenter, COBREXA
 
 ex = quote
-    using COBREXA, Tulip
+    using COBREXA, Tulip, Downloads
     include(joinpath(dirname(pathof(COBREXA)), "..", "test", "data", "test_models.jl"))
     model = test_LP()
```

Comment:
I guess doctests should be able to live for quite a long time with the same files as found now in `data_downloaded.jl`, right?